docker-compose.yml

```
@@ -157,6 +157,23 @@ services:
     networks: [ datalayer-elasticsearch, search-datalayer ]
     mem_limit: 1GB
 
+  pgsync:
+    build: apps/data/pgsync
+    container_name: martletplace_pgsync
+    depends_on:
+      - database
+      - elasticsearch
+    volumes:
+      - ./apps/data/pgsync/config/pgsync.yml:/app/config/pgsync.yml
+    environment:
+      - DB_ENDPOINT=postgres://martletplace:johuigytfrdcv@datalayer:8300/martletplace
+      - ES_ENDPOINT=https://elastic:serxdfcghjfc@elasticsearch:8311
+      - PG_USER=martletplace
+      - PG_PASSWORD=johuigytfrdcv
+    networks:
+      - datalayer-database
+      - datalayer-elasticsearch
```

This should be connected to

```suggestion
      - pgsync-database
      - pgsync-elasticsearch
```

And then you'll need to add one of those to the two databases respectively